src/webhook_service/terraform/lambda.tf

```diff
@@ -159,6 +157,135 @@ module "get_webhook_docker_image" {
   }
 }
 
+####################################
+####################################
+####################################
+# GET /webhooks ####################
+####################################
+####################################
+####################################
+
+module "list_webhooks_lambda" {
+  source  = "terraform-aws-modules/lambda/aws"
+  version = "7.7.0"
+
+  function_name  = local.list_webhooks_function_name_and_ecr_repo_name
+  description    = "AWS Educate TPET ${var.service_hyphen} in ${var.environment}: GET /webhooks"
+  create_package = false
+  timeout        = 30
+
+  ##################
+  # Container Image
+  ##################
+  package_type = "Image"
+  # architectures = ["x86_64"] # or ["arm64"]
+  architectures = ["arm64"]
+  # image_uri     = "public.ecr.aws/lambda/python:3.8"
+  # image_uri     = module.list_emails_docker_image.image_uri
```

Comment:
Remove the commented code

Reply:
Done removing

https://github.com/aws-educate-tw/aws-educate-tpet-backend/blob/SCRUM-157-list-webhooks/src/webhook_service/terraform/lambda.tf